### ISSUE TITLE
app crashes if v.hasVibrate() == false

### ISSUE THREAD

**Issue by Willham12** [mkuczera/react-native-haptic-feedback#72]
```
Caused by java.lang.NoSuchMethodError
No static method createPredefined(I)Landroid/os/VibrationEffect; in class Landroid/os/VibrationEffect;
or its super classes (declaration of 'android.os.VibrationEffect' appears in /system/framework/framework.jar)
com.mkuczera.VibrateWithCreatePredefined.apply (VibrateWithCreatePredefined.java:20)
```

**Comment by Willham12** [comment]
Can close if https://github.com/junina-de/react-native-haptic-feedback/pull/79 merged

**Comment by mkuczera** [comment]
Fixed in 2.1.0, currently in beta to test it.
